src/Components/Alert/Components/Filters/BaseAlertFilter.tsx

```diff
@@ -0,0 +1,73 @@
+import { Checkbox, Column, GridColumns, Spacer, Text } from "@artsy/palette"
+import { FC } from "react"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+import { ShowMore } from "Components/Alert/Components/Filters/ShowMore"
+import { SearchCriteriaAttributeKeys } from "Components/SavedSearchAlert/types"
+
+interface BaseAlertFilterProps {
+  criteriaKey: SearchCriteriaAttributeKeys
+  description?: string
+  expanded?: boolean
+  label: string
+  options: { value: string; name: string }[]
+}
+
+export const BaseAlertFilter: FC<BaseAlertFilterProps> = ({
```

Comment:
I have created this base filter which contains all the logic and can be used like this:

```
    <BaseAlertFilter
      label="Rarity"
      criteriaKey="attributionClass"
      options={ATTRIBUTION_CLASS_OPTIONS}
    />
```

Reply:
I'm happy to hear suggestions for a better name 🙃

Reply:
This abstraction should work for all filters, except colors and price range.

Reply:
**suggestion**: Maybe `QuickMultipleSelectAlertFilter` to communicate that this is a quick/convenient way to spin up a filter as long as the filter:
- shares the same `criteriaKey` (not applicable for "ways to buy")
- is multiple select (checkbox UX)
- assumes that the data persisted to the alert context
- persisted data takes the form of `key: values[]`

Reply:
I've updated the name.